models/games_models.js

```diff
@@ -7,9 +7,33 @@ exports.selectCategories = () => {
 
 };
 
-exports.selectReviews = () => {
-    return db.query(`
-       SELECT * FROM reviews ORDER BY created_at DESC;`).then((result) => {
+exports.selectReviews = (sort_by , category, sort) => {
+    const validSort = ['title', 'designer', 'owner', 'category', 'created_at', 'votes'];
+    
+    if(!validSort.includes(sort_by)){
+        return Promise.reject({status: 400, msg: 'Bad Request'});
+    }
+
+    let queryStr = `SELECT * FROM reviews `;
+    const queryValues = [];
+
+    if(category !== null){
+        queryStr += `WHERE category = $1 ORDER BY ${sort_by} `;
+        queryValues.push(category.replaceAll('_', ' '));
+        if(sort === 'DESC'){
+            queryStr += `DESC;`
+        } else {queryStr += `ASC;`}
+    } else {
+        queryStr += `ORDER BY ${sort_by} `;
+        if(sort === 'DESC'){
+            queryStr += `DESC;`
+        } else {queryStr += `ASC;`}
+    }
```

Comment:
You're repeating some code here, i would separate this to be adding the WHERE clause and queryValues to the if statement, and just do the ORDER BY clause after the if statement